src/ZODB/tests/testSerialize.py

```diff
@@ -139,7 +164,17 @@ def test_funky_datetime_serialization(self):
         # buildout doesn't arrange for the sys.path to be exported,
         # so force it ourselves
         environ = os.environ.copy()
-        environ['PYTHONPATH'] = os.pathsep.join(sys.path)
+        if IS_JYTHON:
+            # Jython 2.7rc2 has a bug; if its Lib directory is
+            # specifically put on the PYTHONPATH, then it doesn't add
+            # it itself, which means it fails to 'import site' because
+            # it can't import '_jythonlib' and the whole process fails
+            # We would use multiprocessing here, but it doesn't exist on jython
+            sys_path = [x for x in sys.path
+                        if not x.endswith('Lib') and x != '__classpath__' and x!= '__pyclasspath__/']
```

Comment:
A link to the Jython bug for the issue would be helpful (e.g., so that we could remove the cruft at some point).